.github/workflows/zowe-cli.yml

```diff
@@ -5,9 +5,15 @@ name: Zowe CLI CI
 
 on:
   push:
-    paths-ignore: 'zowex/**'
+    paths-ignore:
+    - 'zowex/**'
+    - '.github/workflows/rust-cli-publish.yml'
+    - '.github/workflows/rust-cli.yml'
```

Comment:
Nothing important, but curious why not `- '.github/workflows/rust-cli*.yml'` ?

Reply:
clever idea - i can switch over in the next one if that's ok.  im expecting a few more PRs like this to get the binary publishing right